PHP7.1-apache/Dockerfile

```diff
@@ -54,10 +54,21 @@ RUN apt-get update && apt-get install -y lsb-release \
       && cd .. \
       && rm -rf advancecomp-1.17 \
     && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer \
+    \
+    && a2enmod rewrite \
+      && a2enmod ssl  \
+      && mkdir /etc/apache2/ssl \
+      && touch /etc/apache2/ssl/localhost.key \
+      && touch /etc/apache2/ssl/localhost.crt \
+      && openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/apache2/ssl/localhost.key \
+        -subj '/C=CH/ST=Lucerne/L=Sursee/O=w-vision AG/CN=localhost/' -out /etc/apache2/ssl/localhost.crt \
```

Comment:
No quite happy with that, can you change that?

Reply:
What credentials shall I use?

Reply:
Mine :P 

No, just use something generic. Max Mustermans international brother or sister ;)